data/ignition/systemd/units/agent.service

```diff
@@ -0,0 +1,25 @@
+[Service]
+Type=simple
+Restart=always
+RestartSec=3
+StartLimitInterval=0
+Environment=HTTP_PROXY=
+Environment=http_proxy=
+Environment=HTTPS_PROXY=
+Environment=https_proxy=
+Environment=NO_PROXY=
+Environment=no_proxy=
+# TODO: If AUTH_TYPE != none, then PULL_SECRET_TOKEN needs to be updated
+# https://github.com/openshift/assisted-service/blob/master/internal/ignition/ignition.go#L1381
+Environment=PULL_SECRET_TOKEN={{.PullSecretToken}}
+TimeoutStartSec=180
+ExecStartPre=/usr/local/bin/agent-fix-bz1964591 quay.io/edge-infrastructure/assisted-installer-agent:latest
+ExecStartPre=podman run --privileged --rm -v /usr/local/bin:/hostbin quay.io/edge-infrastructure/assisted-installer-agent:latest cp /usr/bin/agent /hostbin
+ExecStart=/usr/local/bin/agent --url {{.ServiceBaseURL}} --infra-env-id {{.infraEnvId}} --agent-version quay.io/edge-infrastructure/assisted-installer-agent:latest --insecure=true
```

Comment:
Yes, specifying ServiceBaseURL through the template will go away once we have AGENT-33/34. This is here in the meantime because it allows me to point it to an assisted-service running in minikube to test a cluster install with the fleeting ISO.